### ISSUE TITLE
Aircrack fails to compile with unstable=true

### ISSUE THREAD

**Issue by aircrack-ng** [aircrack-ng/aircrack-ng#715]
**Reported by flyingstar16 on 19 Jan 2010 10:54 UTC**

When I try to compile with unstable=true I get this error
(note that, as I had previously tried to compile, this output is reduced. Also, output is in Italian, even if understandable in my opinion)
```
claudio@Jarvis:/opt/aircrack-ng$ make sqlite=true unstable=true
make -C src all
make[ingresso nella directory /opt/aircrack-ng/src
make -C osdep
make[2](1]:): ingresso nella directory /opt/aircrack-ng/src/osdep
Building for Linux
make[ingresso nella directory /opt/aircrack-ng/src/osdep
make[3](3]:): .os.Linux  aggiornato.
make[uscita dalla directory /opt/aircrack-ng/src/osdep
make[2](3]:): uscita dalla directory /opt/aircrack-ng/src/osdep
gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1656  -I/usr/local/include -DHAVE_SQLITE -Iinclude   -c -o wesside-ng.o wesside-ng.c
cc1: warnings being treated as errors
wesside-ng.c: In function T.106:
wesside-ng.c:1461: error: dereferencing pointer seq does break strict-aliasing rules
wesside-ng.c:1460: note: initialized from here
make[*** [wesside-ng.o](1]:) Errore 1
make[uscita dalla directory /opt/aircrack-ng/src
make: *** [all](1]:) Errore 2
=====================================================
Ubuntu Lucid Lynx 10.04 alpha 2 x86_64

claudio@Jarvis:/opt/aircrack-ng$ uname -a
Linux Jarvis 2.6.32-10-generic #14-Ubuntu SMP Thu Jan 7 17:38:08 UTC 2010 x86_64 GNU/Linux
=====================================================
claudio@Jarvis:/opt/aircrack-ng$ gcc -v
Using built-in specs.
Target: x86_64-linux-gnu
Configured with: ../src/configure -v --with-pkgversion='Ubuntu 4.4.2-9ubuntu4' --with-bugurl=file:///usr/share/doc/gcc-4.4/README.Bugs --enable-languages=c,c++,fortran,objc,obj-c++ --prefix=/usr --enable-shared --enable-multiarch --enable-linker-build-id --with-system-zlib --libexecdir=/usr/lib --without-included-gettext --enable-threads=posix --with-gxx-include-dir=/usr/include/c++/4.4 --program-suffix=-4.4 --enable-nls --enable-clocale=gnu --enable-libstdcxx-debug --enable-objc-gc --disable-werror --with-arch-32=i486 --with-tune=generic --enable-checking=release --build=x86_64-linux-gnu --host=x86_64-linux-gnu --target=x86_64-linux-gnu
Thread model: posix
gcc version 4.4.3 20100116 (prerelease) (Ubuntu 4.4.2-9ubuntu4) 
=====================================================
claudio@Jarvis:/opt/aircrack-ng$ make -v
GNU Make 3.81
```

**Comment by aircrack-ng** [comment]
**Comment by misterx on 19 Jan 2010 10:54 UTC**

http://seclists.org/nmap-dev/2009/q3/694

http://cellperformance.beyond3d.com/articles/2006/06/understanding-strict-aliasing.html

**Comment by aircrack-ng** [comment]
**Comment by Bill E Ghote  bill e ghote on 19 Jan 2010 10:54 UTC**

Me, too, minus the Italian text, albeit at a different portion of code than the original posting:

gcc -g -W -Wall -Werror -O3 -D_FILE_OFFSET_BITS=64 -D_REVISION=1657  -I/usr/local/include -DHAVE_SQLITE -Iinclude   -c -o wesside-ng.o wesside-ng.c             
cc1: warnings being treated as errors                                           
wesside-ng.c: In function 'T.128':                                              
wesside-ng.c:1461: error: dereferencing pointer 'seq' does break strict-aliasing rules                                                                          
wesside-ng.c:1460: note: initialized from here                                  
make[*** [wesside-ng.o](1]:) Error 1                                             
make[Leaving directory `/home/wdawson/Software/aircrack/build/src'          
make: *** [install](1]:) Error 2

**Comment by aircrack-ng** [comment]
**Comment by misterx on 19 Jan 2010 10:54 UTC**

Fixed in aircrack-ng/aircrack-ng@66fde34ed146457559a313f7b279e3fc68f6df0b.
